src/sentry/static/sentry/app/views/groupDetails/seenBy.jsx

```diff
@@ -31,37 +32,50 @@ const GroupSeenBy = createReactClass({
       return null;
     }
 
-    let seenByNodes = seenBy
-      .filter((user, userIdx) => {
-        return activeUser.id !== user.id;
-      })
-      .map((user, userIdx) => {
-        let title =
-          _.escape(userDisplayName(user)) +
-          '<br/>' +
-          _.escape(moment(user.lastSeen).format('LL'));
-        return (
-          <li key={userIdx}>
-            <Tooltip title={title} tooltipOptions={{html: true}}>
-              <Avatar size={28} user={user} />
-            </Tooltip>
-          </li>
-        );
-      });
-
+    // Note className="seen-by" is required for responsive design
     return (
-      <div className="seen-by">
-        <ul>
-          <li>
-            <Tooltip title={t("People who've viewed this issue")}>
-              <span className="icon-eye" />
-            </Tooltip>
-          </li>
-          {seenByNodes}
-        </ul>
-      </div>
+      <SeenByWrapper className="seen-by">
+        <AvatarList
+          users={seenBy.filter(user => true || activeUser.id !== user.id)}
```

Comment:
is this supposed to filter something?

Reply:
The `true ||` was leftover due to debugging :/ need to make a lint rule for that.

It filters out the current user from the "Seen by" list.